common/scripts/ci/run-sdk-backend-tiger-integration-tests-with-live-backend.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -i
```

Comment:
Should this script be aligned with the K? Or is there that much difference from the integrated tests in KD and in SDK?